index.html

```diff
@@ -159,10 +160,10 @@
 opposite = <span class="literal">true</span>;
 
 <span class="keyword">if</span> (opposite) {
-  number = -<span class="number">42</span>;
+  number = <span class="number">-42</span>;
 }
 
-square = <span class="function"><span class="keyword">function</span><span class="params">(x)</span> {</span>
+square = <span class="function"><span class="keyword">function</span>(<span class="params">x</span>) </span>{
```

Comment:
Are all these little changing due to you updating highlight.js?

I think this PR will be easier to review if you leave out the compiled files. They will be committed when the 1.11.0 release is done.